snippets/rosa-classic-support.adoc

```diff
@@ -0,0 +1,10 @@
+// When including this file, ensure that {FeatureName} is set immediately before
```

Comment:
This unfortunately only occured to me after I merged 🤦‍♀️ This snippet file needs metadata added to it, per: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#text-snippet-file-metadata

@bhardesty can you open another PR to add this? Sorry for the hassle.